meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -31,6 +31,9 @@ SDKMACHINE ?= "x86_64"
 # Generate source mirror tarballs:
 BB_GENERATE_MIRROR_TARBALLS = "1"
 
+# Do not allow building of anything that hasn't been checked beforehand.
+SUPPORTED_RECIPES_CHECK = "fatal"
+
 # Configure images in "development mode".
```

Comment:
I don't think it's good idea to put it here. This should be default value in distro config that user can override in local.conf if he/she wants.